scoresheets/GiveMeAHand.tex

```diff
@@ -1,11 +1,15 @@
 \begin{scorelist}[timelimit=10]
 
-	\scoreheading{Regular Rewards}	
+	\scoreheading{Regular Rewards}
+	\scoreitem{100}{Approach a calling operator}	
+	\scoreitem[5]{50}{Approach the human's hand without contact}
 	\scoreitem[5]{100}{Receive an object from the operator}
+	\scoreitem[5]{50}{Use natural interaction to clarify ambiguous delivery locations}
 	\scoreitem[5]{100}{Place an object at the correct location}
  
 	\scoreheading{Penalties}
-	\penaltyitem[5]{75}{Place an object at the wrong location}
+	\penaltyitem[5]{50}{Initiating unattended interaction (e.g., talking to the open air)}
+	\penaltyitem[5]{75}{Place an object at the wrong location or without user's confirmation}
```

Comment:
`Use natural interaction to clarify ambiguous delivery locations`

As this is already a score item i think we need to specify this in the penalty .
Can we think of some unnatural interaction the robot could use to specify the location?
We could rewrite the penalty like this:

```suggestion
	\penaltyitem[5]{75}{Place an object at the wrong or ambiguous location}
```

Reply:
Ok. What about "Place an object at the wrong location or by guessing the intended location"?